tests/e2e/user-anonymous.spec.js

```diff
@@ -0,0 +1,79 @@
+import { expect, test } from '@playwright/test';
+
+import orchestrator from 'tests/orchestrator';
+
+import { HomePage } from './page-object/home-page';
+import { LoginPage } from './page-object/login-page';
+import { RecentPage } from './page-object/recent-page';
+import { RegisterPage } from './page-object/register-page';
+
+const titleHome = 'TabNews: Conteúdos para quem trabalha com Programação e Tecnologia';
+
+test.beforeAll(async () => {
+  await orchestrator.waitForAllServices();
+  await orchestrator.dropAllTables();
+  await orchestrator.runPendingMigrations();
+});
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/');
+});
+
+test('should be able to login', async ({ page }) => {
+  let homePage = new HomePage(page);
+
+  let title = await homePage.getTitle();
+  await expect(title).toBe(titleHome);
+
+  await homePage.goLogin();
+
+  const loginPage = new LoginPage(page);
+  title = await loginPage.getTitle();
+  await expect(title).toBe('Login · TabNews');
+});
+
+test('should be able to register', async ({ page }) => {
+  let homePage = new HomePage(page);
+
+  let title = await homePage.getTitle();
+  await expect(title).toBe(titleHome);
+
+  await homePage.goRegister();
+
+  const registerPage = new RegisterPage(page);
+  title = await registerPage.getTitle();
+  await expect(title).toBe('Cadastro · TabNews');
+});
+
+test('should be able to see relevant and recent tab', async ({ page }) => {
+  let homePage = new HomePage(page);
+
+  let titleHomePage = await homePage.getTitle();
+  await expect(titleHomePage).toBe(titleHome);
+
+  await homePage.goRecentTab();
+
+  let recentPage = new RecentPage(page);
+  let titleRecentPage = await recentPage.getTitle();
+  await expect(titleRecentPage).toBe('Página 1 · Recentes · TabNews');
+
+  await recentPage.goRelevantTab();
+});
+
+test('should be login like user default', async ({ page }) => {
```

Comment:
Alguns nomes de testes ficaram estranhos. Desse arquivo, acredito que só esse precise ser melhorado, mas do outro arquivo, é preciso rever todos. Isso não é urgente, pode deixar para o final porque assim já garante que todos os testes estarão adequados (tanto os atuais quanto os novos, caso você crie mais nesse PR).

Seguindo o padrão dos outros testes desse arquivo, aqui poderia ser: `test('should be able to login'` .